### PR TITLE
Feature tree development bm

### PR DESCRIPTION
**Pull request by asprtam**
tested tree and it works, still tree postiion is hard coded, it should be generated with world, added comments and png textures that needs to be added in handle level method, which i cant apparently do.

### CODE REVIEW

File: env_gen.cpp [hunk 1]
```diff
@@ -1,6 +1,62 @@
 #include "env_gen.h"
 #include "treeClass.h"
 
+
+void create_neighbours(sprite_params** data, int width, int height)
```
Comment (konstanty-kordas) [inline env_gen.cpp:5]: env_gen::create_neighbours?

Comment (asprtam) [reply env_gen.cpp:5]: yess

**Comment by konstanty-kordas** [comment]
works ok

**Comment by AN0DA** [comment]
@Chl3bak05 Please describe the steps to reproduce the error. It works fine for me

**Pull request merged**